src/test/java/io/javalin/validation/TestValidation.kt

```diff
@@ -160,4 +164,44 @@ class TestValidation {
         assertThat(http.get("/").status, `is`(HttpStatus.EXPECTATION_FAILED_417))
     }
 
+    @Suppress("UNCHECKED_CAST")
+    @Test
+    fun `test validation strict mode`() = TestUtil.test { app, http ->
+        val sillyString = "2"
+
+        JavalinJson.fromJsonMapper = object : FromJsonMapper {
+            override fun <T> map(json: String, targetClass: Class<T>) = sillyString as T
+        }
+
+        val testSchema: Schema = SchemaLoader.load(JSONObject("{\n" +
+                "  \"type\": \"object\",\n" +
+                "  \"required\": [\n" +
+                "    \"test\"\n" +
+                "  ],\n" +
+                "  \"properties\": {\n" +
+                "    \"test\": {\n" +
+                "      \"type\": \"integer\",\n" +
+                "      \"title\": \"The Test Schema\",\n" +
+                "      \"default\": 0\n" +
+                "    }\n" +
+                "  }\n" +
+                "}"))
```

Comment:
Use multi line strings here for better readability. (start and end with triple")

Reply:
Intellij might suggest switching to multi line strings if you press Alt+Enter.

Reply:
Thx, made it in 2c49f23